File: mario.py
# import get_int function from cs50 library
from cs50 import get_int
import sys


def main():

    # height of hashes
    Height = 0
    # hashes block
    hashes = 0
    # row for the hashes
    row = 0

    # check if Hash between 1 and 8
    Height = get_int("the height for the hashes?:")

    if Height > 8 or Height < 1:
        Height = get_int("the height of the hashes?:")

    # a loop through the lines(row)
    while row < Height:
        row += 1
        # print spaces before the hashes
        print(" " * (Height - row), end="")
        # print hashes after the spaces
        print("#" * (row))
        # handle base case
        if row == 0:
            print("hey, i think theres something wrong with me...")


main()

File: mario.c
#include <stdio.h>
#include <cs50.h>

int main(void)
{
    int height = 0;
    do
    {
        height = get_int("height for the hashes: ");
    }
    while (height < 1 || height > 8);

    for (int row = 0; row < height; row++)
    {
        for (int space = height - row - 1; space > 0; space--)
        {
            printf(" ");
        }
        int r = row;

        for (int hash = 0; hash < r + 1; hash++)
        {
            printf("#");
        }
        printf("\n");
    }
}